### ISSUE TITLE
i is not defined - rejectedWith

### ISSUE THREAD

**Issue by YourDeveloperFriend** [shouldjs/promised#3]
When calling `rejectedWith` and the assertion fails, instead of throwing a readable error, it throws `i is not defined`. The offending line appears to be [here](https://github.com/shouldjs/promised/blob/master/index.js#L150).

Example

``` Javascript
Promise.reject('message').should.be.rejectedWith('wrongMessage');
// throws ReferenceError: i is not defined
```

Look for the pending pull request.

**Comment by btd** [comment]
Fixed in 0.0.5 with other errors appeared with 5.0.0

**Comment by YourDeveloperFriend** [comment]
Thanks! Ignore my previous commit, your solution appears better. Thanks a lot!
